stack.py

```diff
@@ -0,0 +1,21 @@
+from linkedlist import LinkedList
+
+
+class Stack:
+    def __init__(self, values=None):
+        self.ll = LinkedList(values)
+        self._length = 0
```

Comment:
Why do you need to track the length here at all?

Reply:
i don't because ._length is a method of a LinkedList